app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -27,7 +27,7 @@ def as_json
           reject_by_default_at: application_choice.reject_by_default_at&.iso8601,
           recruited_at: application_choice.recruited_at,
           candidate: {
-            id: "C#{application_form.candidate.id}",
+            id: application_form.candidate.public_id,
```

Comment:
can you also mirror this in the register API (I'll add a ticket to refactor this so we can pick it up soon)

Reply:
Yes — done!